src/components/Tipseen/__stories__/tipseen.stories.mdx

```diff
@@ -0,0 +1,228 @@
+import Tipseen from "../Tipseen";
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import { createComponentTemplate } from "../../../storybook/functions/create-component-story";
+import TipseenContent from "../TipseenContent";
+import TipseenWizard from "../TipseenWizard";
+import TipseenImage from "../TipseenImage";
+import picture from "./assets/picture.png"
+import Icon from "../../Icon/Icon.jsx";
+import { BoardPrivate, Board } from "../../Icon/Icons"
+import { modifiers } from "./helper.js";
+import { Link } from "../../../storybook/components";
+import { TOOLTIP, ATTENTION_BOX, CHIP } from "../../../storybook/components/related-components/component-description-map";
+import "./tipseen.stories.scss";
+
+<Meta
+  title="Components/Tipseen"
+  component={ Tipseen }
+/>
+
+<!--- Component template -->
+
+export const tipseenTemplate = (args) => {
+  return(
+    <Tipseen
+    // The modifier's purpose is to prevent the tipseen from being displayed when the user scrolls the story upwards / downwards.
+    // Therefore, there is no need to move this prop in your implementations.
+      modifiers={modifiers}
+      position={args.position}
+      content={
+        <TipseenContent
+          isDismissHidden={args.isDismissHidden}
+          title={args.title}
+          children={args.children}
```

Comment:
please pass children props to component by using open tag and close tag syntax:
`<TipseenContent>{args.children}</TipseenContent>`